src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -320,13 +311,11 @@ export default {
   },
   visitType: {
     url: '/new-appointment/choose-visit-type',
-    previous: 'reasonForAppointment',
     next: 'contactInfo',
   },
   appointmentTime: {
     url: '/new-appointment/appointment-time',
     next: 'contactInfo',
-    previous: 'vaFacility',
```

Comment:
I think there are some more previous functions in this file that could be removed